release.ps1

```diff
@@ -24,6 +25,7 @@ Remove-Item -Path ($out + "\BepInEx\plugins\KK_MainGameVR\Plugins") -Force
 Copy-Item -Path ($dir + "\KK\Patcher\*") -Destination ($out + "\BepInEx\patchers\KK_MainGameVR_Patcher") | Out-Null
 Copy-Item -Path ($dir + "\KK\Images\*") -Destination ($out + "\BepInEx\plugins\KK_MainGameVR\Images\") -Force | Out-Null
 Copy-Item -Path ($dir + "\KK\Data\*") -Destination ($out + "\Koikatu_Data") -Recurse  | Out-Null
+Copy-Item -Path ($dir + "\KK\Data\*") -Destination ($out + "\Koikatsu Party_Data") -Recurse  | Out-Null
```

Comment:
```suggestion
```
Changes to release script should go into a separate PR. Also, I think it would be better to have a separate zip for party to not pollute the game directory, since some mods might be checking for the existence of the _Data folders to determine if the game is Party or not.